.markdownlint.yaml

```diff
@@ -0,0 +1,14 @@
+# Default state for all rules
+default: true
+
+# Configure long lines for tables and code blocks
+MD013:
+  code_blocks: false
+  tables: false
+
+# Configure Multiple headings with the same content
+MD024:
+  # Only check sibling headings
+  allow_different_nesting: true
```

Comment:
Does this comment apply to this setting?

Reply:
yes, both of them should be set together. Mainly do changelog purposes.